cmd/kubeadm/app/phases/controlplane/manifests.go

```diff
@@ -195,16 +196,30 @@ func getAPIServerCommand(cfg *kubeadmapi.ClusterConfiguration, localAPIEndpoint
 // Node,RBAC should be fixed in this order at the beginning
 // AlwaysAllow and AlwaysDeny is ignored as they are only for testing
 func getAuthzModes(authzModeExtraArgs string) string {
```

Comment:
This looks like out of the scope of the PR. It may also have some security implications.

Reply:
Sure, let's put this into another PR to discuss.